src/test/osd/osd-scrub-repair.sh

```diff
@@ -32,7 +32,18 @@ function run() {
     local dir=$1
     shift
 
-    export CEPH_MON="127.0.0.1:7107" # git grep '\<7107\>' : there must be only one
+    # Grab a random port from 7109 - 7200, this should maybe be reworked
+    for port in `seq 7109 7200 | shuf` ; do
+        if ! $(nc -z localhost "$port" > /dev/null); then
```

Comment:
This seems racey.  If 2 processes run nc on the same port number and don't see each other, they would both launch a mon with the conflicting port number.

Maybe the caller must be pass a unique port number for each parallel subtest.  There are 38 other tests that are setting CEPH_MON themselves.  Maybe port numbers should be 8000 + (100 * test#) + subtest#.  All other tests are using a unique port in the 7000 -7999 range.  We would fix the 4 tests that use 17108-17110 and 17119.